lib/token.js

```diff
@@ -391,7 +391,7 @@ function buildAuthorizeUrl(sdk, oauthParams) {
  *
  *    Optional:
  *      redirectUri: defaults to window.location.href
- *      scopes: defaults to ['openid', 'email']
+ *      scope: defaults to ['openid', 'email']
```

Comment:
The docs were wrong. I like the idea of the plural `scopes` though, because it's an array. Perhaps we support both, but deprecate `scope`?